packages/docprovider/src/yprovider.ts

```diff
@@ -110,14 +121,32 @@ export class WebSocketProvider implements IDocumentProvider {
     this._awareness.setLocalStateField('user', user.identity);
   }
 
+  private _onConnectionClosed = (event: any): void => {
+    if (event.code === 1003) {
+      console.error('Document provider closed:', event.reason);
+
+      showErrorMessage(
+        'Session expired',
```

Comment:
Could please bring down the `translator` object as new optional constructor option so those strings can be translated.